example_experiments/asreader_wdw_sciq_train_sciq_test.json

```diff
@@ -0,0 +1,30 @@
+// train a model on WhoDidWhat, then on SciQ, and see how well it does on SciQ
+{
+    "model_class": "AttentionSumReader",
+    "model_serialization_prefix": "models/multiple_choice_qa/asreader_wdw_train_sciq_tune_sciq_test",
+    "preferred_backend": "theano",
+    "encoder": {
+        "default": {
+            "type": "bi_gru",
+            "output_dim": 384
+        }
+    },
+    "seq2seq_encoder": {
+        "default": {
+            "type": "bi_gru",
+            "encoder_params": {
+                "output_dim": 384
+            },
+            "wrapper_params": {}
+        }
+    },
+    "optimizer": {"type": "adam",
+                  "clipnorm": 10.0
+                 },
+    "embedding_dropout": 0.0,
+    "patience": 0,
+    "embedding_size": 256,
+    "num_epochs": 5,
+    "train_files": [["/efs/data/dlfa/who_did_what/processed/strict/train.tsv"], ["/efs/data/dlfa/sciq/processed/train_no_passage_with_background_as_passages.tsv"]],
+    "validation_files": ["/efs/data/dlfa/sciq/processed/test_no_passage_with_background_as_passages.tsv"],
```

Comment:
To avoid subtle bugs.  Did you really want to do early stopping on SciQ when you were pre-training on WDW?  To avoid even the possibility of those kinds of errors (which would not be obvious at all in the output), it's best to make sure the user specifies lists in both places if they are doing this.